mimic-iv/concepts/concept_map/chartevents_to_loinc.csv

```diff
@@ -0,0 +1,43 @@
+mimic:220050,Arterial Blood Pressure systolic,skos:exactMatch,loinc:8480-6,Systolic blood pressure,HumanCurated,orcid:0000-0001-8822-1884,orcid:0000-0002-9348-9284,1,
+mimic:220051,Arterial Blood Pressure diastolic,skos:exactMatch,loinc:8462-4,Diastolic blood pressure,HumanCurated,orcid:0000-0001-8822-1884,orcid:0000-0002-9348-9284,1,
+mimic:223761,Temperature Fahrenheit,skos:exactMatch,loinc:8310-5,Body temperature,HumanCurated,orcid:0000-0001-8822-1884,orcid:0000-0002-9348-9284,1,
+mimic:220060,Pulmonary Artery Pressure diastolic,skos:exactMatch,loinc:8385-7,Pulmonary artery Diastolic blood pressure,HumanCurated,orcid:0000-0001-8822-1884,orcid:0000-0002-9348-9284,1,
+mimic:220059,Pulmonary Artery Pressure systolic,skos:exactMatch,loinc:8440-0,Pulmonary artery Systolic blood pressure,HumanCurated,orcid:0000-0001-8822-1884,orcid:0000-0002-9348-9284,1,
+mimic:220061,Pulmonary Artery Pressure mean,skos:exactMatch,loinc:8414-5,Pulmonary artery Mean blood pressure,HumanCurated,orcid:0000-0001-8822-1884,orcid:0000-0002-9348-9284,1,
+mimic:223762,Temperature Celsius,skos:exactMatch,loinc:8310-5,Body temperature,HumanCurated,orcid:0000-0001-8822-1884,orcid:0000-0002-9348-9284,1,
+mimic:225312,ART BP Mean,skos:exactMatch,loinc:8478-0,Mean blood pressure,HumanCurated,orcid:0000-0001-8822-1884,orcid:0000-0002-9348-9284,1,
+mimic:225309,ART BP Systolic,skos:exactMatch,loinc:8480-6,Systolic blood pressure,HumanCurated,orcid:0000-0001-8822-1884,orcid:0000-0002-9348-9284,1,
+mimic:225310,ART BP Diastolic,skos:exactMatch,loinc:8462-4,Diastolic blood pressure,HumanCurated,orcid:0000-0001-8822-1884,orcid:0000-0002-9348-9284,1,
+mimic:226329,Blood Temperature CCO (C),skos:exactMatch,loinc:60834-9,Blood temperature,HumanCurated,orcid:0000-0001-8822-1884,orcid:0000-0002-9348-9284,1,
+mimic:226531,Admission Weight (lbs.),skos:exactMatch,loinc:29463-7,Body weight,HumanCurated,orcid:0000-0001-8822-1884,orcid:0000-0002-9348-9284,1,
+mimic:228640,EtCO2,skos:exactMatch,loinc:19891-1,Carbon dioxide [Partial pressure] in Exhaled gas --at end expiration,HumanCurated,orcid:0000-0001-8822-1884,orcid:0000-0002-9348-9284,1,
+mimic:226512,Admission Weight (Kg),skos:exactMatch,loinc:29463-7,Body weight,HumanCurated,orcid:0000-0001-8822-1884,orcid:0000-0002-9348-9284,1,
+mimic:227632,Arctic Sun/Alsius Temp #1 C,skos:closeMatch,loinc:8310-5,Body temperature,HumanCurated,orcid:0000-0001-8822-1884,orcid:0000-0002-9348-9284,0.8,
+mimic:227634,Arctic Sun/Alsius Temp #2 C,skos:closeMatch,loinc:8310-5,Body temperature,HumanCurated,orcid:0000-0001-8822-1884,orcid:0000-0002-9348-9284,0.8,
+mimic:226707,Height,skos:exactMatch,loinc:8302-2,Body height,HumanCurated,orcid:0000-0001-8822-1884,orcid:0000-0002-9348-9284,1,
+mimic:226730,Height (cm),skos:exactMatch,loinc:8302-2,Body height,HumanCurated,orcid:0000-0001-8822-1884,orcid:0000-0002-9348-9284,1,
```

Comment:
I made some comments in https://github.com/MIT-LCP/mimic-code/pull/1418 so I don't repeat them here, but I am just nothing that there is something odd about using exactMatch for unit-restriced terms mapped to unit-unrestricted terms. Not necessarily wrong, just.. Should be considered!

Reply:
Thank you @matentzn! Do you think "skos:closeMatch" is a better `predicate_id` in this case?